# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fenced-frame/sandbox-mandatory-flags.https.html.ini
# Commit: 8339bdf8fcc8
# Full Hash: 8339bdf8fcc85604672251b5c26800f2638e69d2
# Author: Cosmin Sabou <csabou@mozilla.com>
# Date: 2023-04-29 09:20:24
# Description:
#   Bug 1829504 - Mark fenced-frame tests as intermittent on win and linux debug. r=aryx
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D176769
# ==============================================================================

diff -r 48f93c057e50 -r 8339bdf8fcc8 testing/web-platform/meta/fenced-frame/sandbox-mandatory-flags.https.html.ini
--- a/testing/web-platform/meta/fenced-frame/sandbox-mandatory-flags.https.html.ini	Fri Apr 28 17:51:25 2023 +0000
+++ b/testing/web-platform/meta/fenced-frame/sandbox-mandatory-flags.https.html.ini	Fri Apr 28 17:26:15 2023 +0000
@@ -1,7 +1,7 @@
 [sandbox-mandatory-flags.https.html]
   expected:
-    if (os == "win") and debug and (processor == "x86_64") and not swgl: [TIMEOUT, CRASH]
-    if (os == "linux") and debug and fission and not swgl: [TIMEOUT, CRASH]
+    if (os == "win") and debug: [TIMEOUT, CRASH]
+    if (os == "linux") and debug: [TIMEOUT, CRASH]
     TIMEOUT
   [Sandboxed Iframe with mandatory flags can load a fenced frame.]
     expected: TIMEOUT
